Auto update failing for vulkan-extension-layer

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the vulkan-extension-layer package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating vulkan-extension-layer [Current version: 1.3.271]
INFO: package being updated to 1.3.274.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 1133k    0 1133k    0     0  1655k      0 --:--:-- --:--:-- --:--:-- 1655k

You are about to commit these changes:

--------------------
diff --git a/packages/vulkan-extension-layer/build.sh b/packages/vulkan-extension-layer/build.sh
index 379ad2ec0..707f42823 100644
--- a/packages/vulkan-extension-layer/build.sh
+++ b/packages/vulkan-extension-layer/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/KhronosGroup/Vulkan-ExtensionLayer
 TERMUX_PKG_DESCRIPTION="Vulkan Extension Layer"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.3.271"
+TERMUX_PKG_VERSION="1.3.274"
 TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/Vulkan-ExtensionLayer/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=bb4acee7d7aba468417fc6efb5649ecf4275772b1b7d5b1fb2ae4f85d765b93c
+TERMUX_PKG_SHA256=100ccf35f4b54afe9601374f2aadee8b6dc24cde7273b31d1432192a0c52c2bc
 TERMUX_PKG_DEPENDS="libc++, vulkan-loader"
 TERMUX_PKG_BUILD_DEPENDS="vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-loader-generic (=${TERMUX_PKG_VERSION}), vulkan-utility-libraries (=${TERMUX_PKG_VERSION})"
 TERMUX_PKG_ANTI_BUILD_DEPENDS="vulkan-loader"
--------------------

bump(main/vulkan-extension-layer): 1.3.274


Not committing to Git!

INFO: Trying to build package.
INFO: Package vulkan-extension-layer exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building vulkan-extension-layer for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0   224k      0 --:--:-- --:--:-- --:--:--  229k
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1416      0 --:--:-- --:--:-- --:--:--  1414
gpg: Signature made Thu 21 Dec 2023 12:40:01 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  20796      0 --:--:-- --:--:-- --:--:-- 20780
100 13320  100 13320    0     0  20793      0 --:--:-- --:--:-- --:--:-- 20780
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1554      0 --:--:-- --:--:-- --:--:--  1557
gpg: Signature made Wed 20 Dec 2023 06:15:30 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  59302      0 --:--:-- --:--:-- --:--:-- 59366
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1542      0 --:--:-- --:--:-- --:--:--  1542
gpg: Signature made Tue 19 Dec 2023 08:56:19 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libc++@26b if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency vulkan-headers@1.3.274 if necessary...
Found vulkan-headers in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/vulkan-headers/vulkan-headers_1.3.274_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  2 1252k    2 30700    0     0  29980      0  0:00:42  0:00:01  0:00:41 30009
100 1252k  100 1252k    0     0   697k      0  0:00:01  0:00:01 --:--:--  697k
extracting vulkan-headers to /home/builder/.termux-build/_cache-all...
Downloading dependency libandroid-support@28-3 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xorg-util-macros@1.20.0 if necessary...
Found xorg-util-macros in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorg-util-macros to /home/builder/.termux-build/_cache-all...
Downloading dependency xorgproto@2023.2 if necessary...
Found xorgproto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorgproto to /home/builder/.termux-build/_cache-all...
Downloading dependency libxau@1.0.11 if necessary...
Found libxau in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxau to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxdmcp@1.1.4 if necessary...
Found libxdmcp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxdmcp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xcb-proto@1.16.0 if necessary...
Found xcb-proto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xcb-proto to /home/builder/.termux-build/_cache-all...
Downloading dependency python-xcbgen@1.16.0 if necessary...
Found python-xcbgen in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-xcbgen to /home/builder/.termux-build/_cache-all...
Downloading dependency libxcb@1.16 if necessary...
Found libxcb in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxcb to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xtrans@1.5.0 if necessary...
Found xtrans in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xtrans to /home/builder/.termux-build/_cache-all...
Downloading dependency libx11@1.8.7 if necessary...
Found libx11 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libx11 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxext@1.3.5 if necessary...
Found libxext in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxext to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrender@0.9.11 if necessary...
Found libxrender in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxrender to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrandr@1.5.4 if necessary...
Found libxrandr in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxrandr/libxrandr_1.5.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 21824  100 21824    0     0   345k      0 --:--:-- --:--:-- --:--:--  349k
extracting libxrandr to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency vulkan-loader-generic@1.3.271 if necessary...
Found vulkan-loader-generic in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/vulkan-loader-generic/vulkan-loader-generic_1.3.271_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 28  105k   28 30704    0     0  32761      0  0:00:03 --:--:--  0:00:03 32733
100  105k  100  105k    0     0  97191      0  0:00:01  0:00:01 --:--:-- 97209
extracting vulkan-loader-generic to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency vulkan-utility-libraries@1.3.271 if necessary...
Download of vulkan-utility-libraries@1.3.271 from https://packages-cf.termux.dev/apt/termux-main failed, building instead
termux - building vulkan-utility-libraries for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0  22055      0 --:--:-- --:--:-- --:--:-- 22036
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1404      0 --:--:-- --:--:-- --:--:--  1402
100   833  100   833    0     0   1404      0 --:--:-- --:--:-- --:--:--  1402
gpg: Signature made Thu 21 Dec 2023 12:40:01 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  24773      0 --:--:-- --:--:-- --:--:-- 24804
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0  13075      0 --:--:-- --:--:-- --:--:-- 13222
gpg: Signature made Wed 20 Dec 2023 06:15:30 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0   220k      0 --:--:-- --:--:-- --:--:--  220k
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1542      0 --:--:-- --:--:-- --:--:--  1542
gpg: Signature made Tue 19 Dec 2023 08:56:19 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libc++@26b if necessary...
Skipping already built dependency libc++@26b
Downloading dependency vulkan-headers@1.3.274 if necessary...
Skipping already built dependency vulkan-headers@1.3.274
Downloading dependency libandroid-support@28-3 if necessary...
Skipping already built dependency libandroid-support@28-3
Downloading dependency xorg-util-macros@1.20.0 if necessary...
Skipping already built dependency xorg-util-macros@1.20.0
Downloading dependency xorgproto@2023.2 if necessary...
Skipping already built dependency xorgproto@2023.2
Downloading dependency libxau@1.0.11 if necessary...
Skipping already built dependency libxau@1.0.11
Downloading dependency libxdmcp@1.1.4 if necessary...
Skipping already built dependency libxdmcp@1.1.4
Downloading dependency xcb-proto@1.16.0 if necessary...
Skipping already built dependency xcb-proto@1.16.0
Downloading dependency python-xcbgen@1.16.0 if necessary...
Skipping already built dependency python-xcbgen@1.16.0
Downloading dependency libxcb@1.16 if necessary...
Skipping already built dependency libxcb@1.16
Downloading dependency xtrans@1.5.0 if necessary...
Skipping already built dependency xtrans@1.5.0
Downloading dependency libx11@1.8.7 if necessary...
Skipping already built dependency libx11@1.8.7
Downloading dependency libxext@1.3.5 if necessary...
Skipping already built dependency libxext@1.3.5
Downloading dependency libxrender@0.9.11 if necessary...
Skipping already built dependency libxrender@0.9.11
Downloading dependency libxrandr@1.5.4 if necessary...
Skipping already built dependency libxrandr@1.5.4
Downloading dependency vulkan-loader-generic@1.3.271 if necessary...
Skipping already built dependency vulkan-loader-generic@1.3.271
Downloading https://github.com/KhronosGroup/Vulkan-Utility-Libraries/archive/refs/tags/v1.3.271.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  162k    0  162k    0     0   300k      0 --:--:-- --:--:-- --:--:--  300k
Applying patch: 0001-do-not-build-for-android.patch
Downloading https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-linux.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

 15  100k   15 16375    0     0  37973      0  0:00:02 --:--:--  0:00:02 37973
100  100k  100  100k    0     0   224k      0 --:--:-- --:--:-- --:--:-- 4711k
Archive:  /home/builder/.termux-build/vulkan-utility-libraries/tmp/ninja-1.10.2.zip
  inflating: /home/builder/.termux-build/_cache/ninja-1.10.2/ninja  
-- The CXX compiler identification is Clang 17.0.2
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/builder/.termux-build/_cache/android-r26b-api-24-v0/bin/aarch64-linux-android-clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring done (0.2s)
-- Generating done (0.0s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_ANDROID_STANDALONE_TOOLCHAIN
    CMAKE_C_FLAGS
    CMAKE_FIND_ROOT_PATH_MODE_LIBRARY
    CMAKE_TERMUX_TOOL_C_COMPILER
    CMAKE_USE_SYSTEM_LIBRARIES
    DOXYGEN_EXECUTABLE


-- Build files have been written to: /home/builder/.termux-build/vulkan-utility-libraries/build
[1/5] Building CXX object src/layer/CMakeFiles/VulkanLayerSettings.dir/vk_layer_settings_helper.cpp.o
FAILED: src/layer/CMakeFiles/VulkanLayerSettings.dir/vk_layer_settings_helper.cpp.o 
/home/builder/.termux-build/_cache/android-r26b-api-24-v0/bin/aarch64-linux-android-clang++ -DVK_ENABLE_BETA_EXTENSIONS -I/home/builder/.termux-build/vulkan-utility-libraries/src/include -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wpedantic -Wunreachable-code -Wunused-function -Wall -Wextra -Wpointer-arith -Wextra-semi -Wunreachable-code-return -Wconversion -Wimplicit-fallthrough -Wstring-conversion -MD -MT src/layer/CMakeFiles/VulkanLayerSettings.dir/vk_layer_settings_helper.cpp.o -MF src/layer/CMakeFiles/VulkanLayerSettings.dir/vk_layer_settings_helper.cpp.o.d -o src/layer/CMakeFiles/VulkanLayerSettings.dir/vk_layer_settings_helper.cpp.o -c /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings_helper.cpp
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings_helper.cpp:9:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.hpp:12:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.h:16:
/home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings_ext.h:19:9: warning: 'VK_EXT_LAYER_SETTINGS_SPEC_VERSION' macro redefined [-Wmacro-redefined]
#define VK_EXT_LAYER_SETTINGS_SPEC_VERSION 1
        ^
/data/data/com.termux/files/usr/include/vulkan/vulkan_core.h:18479:9: note: previous definition is here
#define VK_EXT_LAYER_SETTINGS_SPEC_VERSION 2
        ^
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings_helper.cpp:9:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.hpp:12:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.h:16:
/home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings_ext.h:26:14: error: redefinition of 'VkLayerSettingTypeEXT'
typedef enum VkLayerSettingTypeEXT {
             ^
/data/data/com.termux/files/usr/include/vulkan/vulkan_core.h:18482:14: note: previous definition is here
typedef enum VkLayerSettingTypeEXT {
             ^
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings_helper.cpp:9:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.hpp:12:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.h:16:
/home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings_ext.h:37:16: error: redefinition of 'VkLayerSettingEXT'
typedef struct VkLayerSettingEXT {
               ^
/data/data/com.termux/files/usr/include/vulkan/vulkan_core.h:18493:16: note: previous definition is here
typedef struct VkLayerSettingEXT {
               ^
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings_helper.cpp:9:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.hpp:12:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.h:16:
/home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings_ext.h:45:16: error: redefinition of 'VkLayerSettingsCreateInfoEXT'
typedef struct VkLayerSettingsCreateInfoEXT {
               ^
/data/data/com.termux/files/usr/include/vulkan/vulkan_core.h:18501:16: note: previous definition is here
typedef struct VkLayerSettingsCreateInfoEXT {
               ^
1 warning and 3 errors generated.
[2/5] Building CXX object src/layer/CMakeFiles/VulkanLayerSettings.dir/vk_layer_settings.cpp.o
FAILED: src/layer/CMakeFiles/VulkanLayerSettings.dir/vk_layer_settings.cpp.o 
/home/builder/.termux-build/_cache/android-r26b-api-24-v0/bin/aarch64-linux-android-clang++ -DVK_ENABLE_BETA_EXTENSIONS -I/home/builder/.termux-build/vulkan-utility-libraries/src/include -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wpedantic -Wunreachable-code -Wunused-function -Wall -Wextra -Wpointer-arith -Wextra-semi -Wunreachable-code-return -Wconversion -Wimplicit-fallthrough -Wstring-conversion -MD -MT src/layer/CMakeFiles/VulkanLayerSettings.dir/vk_layer_settings.cpp.o -MF src/layer/CMakeFiles/VulkanLayerSettings.dir/vk_layer_settings.cpp.o.d -o src/layer/CMakeFiles/VulkanLayerSettings.dir/vk_layer_settings.cpp.o -c /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:9:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.h:16:
/home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings_ext.h:19:9: warning: 'VK_EXT_LAYER_SETTINGS_SPEC_VERSION' macro redefined [-Wmacro-redefined]
#define VK_EXT_LAYER_SETTINGS_SPEC_VERSION 1
        ^
/data/data/com.termux/files/usr/include/vulkan/vulkan_core.h:18479:9: note: previous definition is here
#define VK_EXT_LAYER_SETTINGS_SPEC_VERSION 2
        ^
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:9:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.h:16:
/home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings_ext.h:26:14: error: redefinition of 'VkLayerSettingTypeEXT'
typedef enum VkLayerSettingTypeEXT {
             ^
/data/data/com.termux/files/usr/include/vulkan/vulkan_core.h:18482:14: note: previous definition is here
typedef enum VkLayerSettingTypeEXT {
             ^
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:9:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.h:16:
/home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings_ext.h:37:16: error: redefinition of 'VkLayerSettingEXT'
typedef struct VkLayerSettingEXT {
               ^
/data/data/com.termux/files/usr/include/vulkan/vulkan_core.h:18493:16: note: previous definition is here
typedef struct VkLayerSettingEXT {
               ^
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:9:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.h:16:
/home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings_ext.h:45:16: error: redefinition of 'VkLayerSettingsCreateInfoEXT'
typedef struct VkLayerSettingsCreateInfoEXT {
               ^
/data/data/com.termux/files/usr/include/vulkan/vulkan_core.h:18501:16: note: previous definition is here
typedef struct VkLayerSettingsCreateInfoEXT {
               ^
/home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:150:53: error: no member named 'count' in 'VkLayerSettingEXT'
                    if (*pValueCount < api_setting->count) {
                                       ~~~~~~~~~~~  ^
/home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:153:84: error: no member named 'count' in 'VkLayerSettingEXT'
                    const std::uint32_t size = std::min(*pValueCount, api_setting->count);
                                                                      ~~~~~~~~~~~  ^
/home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:157:49: error: no member named 'count' in 'VkLayerSettingEXT'
                    *pValueCount = api_setting->count;
                                   ~~~~~~~~~~~  ^
/home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:195:53: error: no member named 'count' in 'VkLayerSettingEXT'
                    if (*pValueCount < api_setting->count) {
                                       ~~~~~~~~~~~  ^
/home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:198:84: error: no member named 'count' in 'VkLayerSettingEXT'
                    const std::uint32_t size = std::min(*pValueCount, api_setting->count);
                                                                      ~~~~~~~~~~~  ^
/home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:202:49: error: no member named 'count' in 'VkLayerSettingEXT'
                    *pValueCount = api_setting->count;
                                   ~~~~~~~~~~~  ^
/home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:249:53: error: no member named 'count' in 'VkLayerSettingEXT'
                    if (*pValueCount < api_setting->count) {
                                       ~~~~~~~~~~~  ^
/home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:252:84: error: no member named 'count' in 'VkLayerSettingEXT'
                    const std::uint32_t size = std::min(*pValueCount, api_setting->count);
                                                                      ~~~~~~~~~~~  ^
/home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:256:49: error: no member named 'count' in 'VkLayerSettingEXT'
                    *pValueCount = api_setting->count;
                                   ~~~~~~~~~~~  ^
/home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:303:53: error: no member named 'count' in 'VkLayerSettingEXT'
                    if (*pValueCount < api_setting->count) {
                                       ~~~~~~~~~~~  ^
/home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:306:79: error: no member named 'count' in 'VkLayerSettingEXT'
                    const uint32_t size = std::min(*pValueCount, api_setting->count);
                                                                 ~~~~~~~~~~~  ^
/home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:310:49: error: no member named 'count' in 'VkLayerSettingEXT'
                    *pValueCount = api_setting->count;
                                   ~~~~~~~~~~~  ^
/home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:357:53: error: no member named 'count' in 'VkLayerSettingEXT'
                    if (*pValueCount < api_setting->count) {
                                       ~~~~~~~~~~~  ^
/home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:360:84: error: no member named 'count' in 'VkLayerSettingEXT'
                    const std::uint32_t size = std::min(*pValueCount, api_setting->count);
                                                                      ~~~~~~~~~~~  ^
/home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:364:49: error: no member named 'count' in 'VkLayerSettingEXT'
                    *pValueCount = api_setting->count;
                                   ~~~~~~~~~~~  ^
/home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/vk_layer_settings.cpp:402:53: error: no member named 'count' in 'VkLayerSettingEXT'
                    if (*pValueCount < api_setting->count) {
                                       ~~~~~~~~~~~  ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
1 warning and 20 errors generated.
[3/5] Building CXX object src/layer/CMakeFiles/VulkanLayerSettings.dir/layer_settings_manager.cpp.o
FAILED: src/layer/CMakeFiles/VulkanLayerSettings.dir/layer_settings_manager.cpp.o 
/home/builder/.termux-build/_cache/android-r26b-api-24-v0/bin/aarch64-linux-android-clang++ -DVK_ENABLE_BETA_EXTENSIONS -I/home/builder/.termux-build/vulkan-utility-libraries/src/include -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wpedantic -Wunreachable-code -Wunused-function -Wall -Wextra -Wpointer-arith -Wextra-semi -Wunreachable-code-return -Wconversion -Wimplicit-fallthrough -Wstring-conversion -MD -MT src/layer/CMakeFiles/VulkanLayerSettings.dir/layer_settings_manager.cpp.o -MF src/layer/CMakeFiles/VulkanLayerSettings.dir/layer_settings_manager.cpp.o.d -o src/layer/CMakeFiles/VulkanLayerSettings.dir/layer_settings_manager.cpp.o -c /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_manager.cpp
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_manager.cpp:9:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_manager.hpp:11:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.h:16:
/home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings_ext.h:19:9: warning: 'VK_EXT_LAYER_SETTINGS_SPEC_VERSION' macro redefined [-Wmacro-redefined]
#define VK_EXT_LAYER_SETTINGS_SPEC_VERSION 1
        ^
/data/data/com.termux/files/usr/include/vulkan/vulkan_core.h:18479:9: note: previous definition is here
#define VK_EXT_LAYER_SETTINGS_SPEC_VERSION 2
        ^
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_manager.cpp:9:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_manager.hpp:11:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.h:16:
/home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings_ext.h:26:14: error: redefinition of 'VkLayerSettingTypeEXT'
typedef enum VkLayerSettingTypeEXT {
             ^
/data/data/com.termux/files/usr/include/vulkan/vulkan_core.h:18482:14: note: previous definition is here
typedef enum VkLayerSettingTypeEXT {
             ^
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_manager.cpp:9:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_manager.hpp:11:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.h:16:
/home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings_ext.h:37:16: error: redefinition of 'VkLayerSettingEXT'
typedef struct VkLayerSettingEXT {
               ^
/data/data/com.termux/files/usr/include/vulkan/vulkan_core.h:18493:16: note: previous definition is here
typedef struct VkLayerSettingEXT {
               ^
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_manager.cpp:9:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_manager.hpp:11:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.h:16:
/home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings_ext.h:45:16: error: redefinition of 'VkLayerSettingsCreateInfoEXT'
typedef struct VkLayerSettingsCreateInfoEXT {
               ^
/data/data/com.termux/files/usr/include/vulkan/vulkan_core.h:18501:16: note: previous definition is here
typedef struct VkLayerSettingsCreateInfoEXT {
               ^
1 warning and 3 errors generated.
[4/5] Building CXX object src/layer/CMakeFiles/VulkanLayerSettings.dir/layer_settings_util.cpp.o
FAILED: src/layer/CMakeFiles/VulkanLayerSettings.dir/layer_settings_util.cpp.o 
/home/builder/.termux-build/_cache/android-r26b-api-24-v0/bin/aarch64-linux-android-clang++ -DVK_ENABLE_BETA_EXTENSIONS -I/home/builder/.termux-build/vulkan-utility-libraries/src/include -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wpedantic -Wunreachable-code -Wunused-function -Wall -Wextra -Wpointer-arith -Wextra-semi -Wunreachable-code-return -Wconversion -Wimplicit-fallthrough -Wstring-conversion -MD -MT src/layer/CMakeFiles/VulkanLayerSettings.dir/layer_settings_util.cpp.o -MF src/layer/CMakeFiles/VulkanLayerSettings.dir/layer_settings_util.cpp.o.d -o src/layer/CMakeFiles/VulkanLayerSettings.dir/layer_settings_util.cpp.o -c /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_util.cpp
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_util.cpp:9:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_util.hpp:11:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.h:16:
/home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings_ext.h:19:9: warning: 'VK_EXT_LAYER_SETTINGS_SPEC_VERSION' macro redefined [-Wmacro-redefined]
#define VK_EXT_LAYER_SETTINGS_SPEC_VERSION 1
        ^
/data/data/com.termux/files/usr/include/vulkan/vulkan_core.h:18479:9: note: previous definition is here
#define VK_EXT_LAYER_SETTINGS_SPEC_VERSION 2
        ^
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_util.cpp:9:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_util.hpp:11:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.h:16:
/home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings_ext.h:26:14: error: redefinition of 'VkLayerSettingTypeEXT'
typedef enum VkLayerSettingTypeEXT {
             ^
/data/data/com.termux/files/usr/include/vulkan/vulkan_core.h:18482:14: note: previous definition is here
typedef enum VkLayerSettingTypeEXT {
             ^
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_util.cpp:9:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_util.hpp:11:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.h:16:
/home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings_ext.h:37:16: error: redefinition of 'VkLayerSettingEXT'
typedef struct VkLayerSettingEXT {
               ^
/data/data/com.termux/files/usr/include/vulkan/vulkan_core.h:18493:16: note: previous definition is here
typedef struct VkLayerSettingEXT {
               ^
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_util.cpp:9:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/src/layer/layer_settings_util.hpp:11:
In file included from /home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings.h:16:
/home/builder/.termux-build/vulkan-utility-libraries/src/include/vulkan/layer/vk_layer_settings_ext.h:45:16: error: redefinition of 'VkLayerSettingsCreateInfoEXT'
typedef struct VkLayerSettingsCreateInfoEXT {
               ^
/data/data/com.termux/files/usr/include/vulkan/vulkan_core.h:18501:16: note: previous definition is here
typedef struct VkLayerSettingsCreateInfoEXT {
               ^
1 warning and 3 errors generated.
ninja: build stopped: subcommand failed.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2023-12-21 06:13:30 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/7284614541">7284614541</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/scripts/bin/update-packages
+++ b/scripts/bin/update-packages
@@ -246,10 +246,11 @@ _gh_check_issue_exists() {
 			_CACHED_ISSUE_TITLES+=("'${title}'") # An extra quote ('') is added to avoid false positive matches.
 		done <<<"$(
 			gh issue list \
+				--limit 10000 \
 				--label "auto update failing" --label "bot" \
 				--state open \
 				--search "Auto update failing for in:title type:issue" \
-				--json title | jq -r '.[] | .title'
+				--json title | jq -r '.[] | .title' | sort | uniq
 		)"
 	fi
 	# shellcheck disable=SC2076 # We want literal match here, not regex based.

